torch/_higher_order_ops/cond.py

```diff
@@ -44,43 +44,35 @@ class UnsupportedAliasMutationException(RuntimeError):
 
 def cond(pred, true_fn, false_fn, operands):
     r"""
-    Conditionally applies ``true_fn`` or ``false_fn``.
+    Conditionally applies `true_fn` or `false_fn`.
```

Comment:
Please keep the prototype warning. cond is not going out of prototype until we do a feature review for it (or for export) to promote to beta

Reply:
added back.